Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -735,11 +738,35 @@ def get_mapping_fields_command():
     indexes = FETCH_INDEX.split(',')
     elastic_mapping = {}
     for index in indexes:
-        res = requests.get(SERVER + '/' + index + '/_mapping', auth=(USERNAME, PASSWORD), verify=INSECURE)
-        my_map = res.json()[index]['mappings']['properties']
-        elastic_mapping[index] = {"_id": "doc_id", "_index": index}
-        elastic_mapping[index]["_source"] = parse_subtree(my_map)
-    demisto.results(elastic_mapping)
+        if index == '':
+            res = requests.get(SERVER + '/_mapping', auth=(USERNAME, PASSWORD), verify=INSECURE)
+        else:
+            res = requests.get(SERVER + '/' + index + '/_mapping', auth=(USERNAME, PASSWORD), verify=INSECURE)
+        res_json = res.json()
+
+        # To get mappings for all data streams and indices in a cluster,
+        # use _all or * for <target> or omit the <target> parameter - from Elastic API
+        if index in ['*', '_all', '']:
+            for key in res_json.keys():
```

Comment:
The `.keys()` is redundant, but note that you can use `.values()` which will return a reference to the dictionary in the item.
```suggestion
            for key in res_json:
```